src/main/java/net/fabricmc/fabric/impl/container/SyncIdProvider.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2016, 2017, 2018 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.container;
+
+/**
+ * This is a interface that is present on a ServerPlayerEntity, it allows access to the sync id.
+ */
+public interface SyncIdProvider {
+
+	/**
+	 * @return the current sync id of the player
+	 */
+	int getSyncId();
```

Comment:
Is this method used anywhere? I can't see any obvious uses, so just wondering if it's safe to remove.

Reply:
No its not, I added it because it may be useful, it can go.